rest/config.go

```diff
@@ -197,6 +197,7 @@ type DbConfig struct {
 	BucketOpTimeoutMs         *uint32                        `json:"bucket_op_timeout_ms,omitempty"`         // How long bucket ops should block returning "operation timed out". If nil, uses GoCB default.  GoCB buckets only.
 	DeltaSync                 *DeltaSyncConfig               `json:"delta_sync,omitempty"`                   // Config for delta sync
 	CompactIntervalDays       *float32                       `json:"compact_interval_days,omitempty"`        //Interval in days between compaction is automatically ran - 0 means don't run
+	HttpOnlyCookies           *bool                          `json:"http_only_cookies"`                      // HTTP only cookies
```

Comment:
If the default value for this is false, we can probably remove the pointer - it simplifies some of the config setup code.